pkg/mock/mockoption/mockoption.go

```diff
@@ -238,3 +242,11 @@ func (mso *ScheduleOptions) IsLocationReplacementEnabled() bool {
 func (mso *ScheduleOptions) IsNamespaceRelocationEnabled() bool {
 	return !mso.DisableNamespaceRelocation
 }
+
+// GetLeaderScheduleKind is to get leader schedule kind
+func (mso *ScheduleOptions) GetLeaderScheduleKind() core.LeaderScheduleKind {
+	if mso.LeaderScoreStrategy == core.LeaderScheduleKind(core.BySize).String() {
```

Comment:
There is no need to use `core.LeaderScheduleKind(core.BySize)`, the others are the same.